src/app/payment/paymentMethod/MasterpassPaymentMethod.tsx

```diff
@@ -20,8 +20,14 @@ const MasterpassPaymentMethod: FunctionComponent<MasterpassPaymentMethodProps &
         },
     }), [initializePayment]);
 
+    const formatLocale = useCallback((localeLanguage: string) => {
```

Comment:
`useCallback` isn't required for functions imported from modules.

Typically we only need it when functions are going in or out of props, and only then to preserve reference equality: https://reactjs.org/docs/hooks-reference.html#usecallback